server/controllers/notification.js

```diff
@@ -0,0 +1,109 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+const notifications = {
+  newRequest: {
+    title: "New Request",
+    message: "You recieved a new request from ",
+  },
+  message: {
+    title: "New Message",
+    message: "You received a new message from ",
+  },
+  requestResponse: {
+    title: "Response Received",
+    message: "You received a response from ",
+  },
+};
+
+// @route POST /notification/:id
+// @desc create notification
+// @access Private
+exports.createNotification = asyncHandler(async (req, res) => {
+  const { sender, type } = req.query;
```

Comment:
You may want to send this information using the request body, let's reserve query parameters (as they're optional) only when we want to use the information to query a database, or maybe to filter results

Reply:
Request body implemented.